src/eko/output.py

```diff
@@ -3,31 +3,32 @@
     This file contains the output management
 """
 import logging
+import copy
 
 import numpy as np
-from yaml import dump
+import yaml
 
 import eko.interpolation as interpolation
-from eko.evolution_operator import apply_PDF_to_operator
+from eko.evolution_operator import OperatorMember, PhysicalOperator
 
 logger = logging.getLogger(__name__)
 
 
 class Output(dict):
     """
         Wrapper for the output to help with application
-        to PDFs and dumping to file
+        to PDFs and dumping to file.
```

Comment:
I already commented about this but I think it is quite important to have something that tells the user that they are doing the wrong thing. Since you are using `yaml` you can add at the end of the module:

```
def this_is_wrong(dumper, data):
    raise Exception("Very wrong, use to_yaml instead")

yaml.add_representer(Test, this_is_wrong)
```

this is a minimal change that can avoid inevitable long phone calls where people discuss what `yaml.dump` should or not do.

Of course, instead of an exception a warning would be also ok (and maybe even better). Anything so that when the user saves their things wrong at least is not silently wrong...

Reply:
we will fix in the future - I took a note